apps/server/src/services/ElasticSearchClient.ts

```diff
@@ -316,13 +320,17 @@ export async function getInstitution(id: string): Promise<CachedInstitution> {
 
 export async function getRecommendedInstitutions({
   jobTypes,
+  aggregatorOverride,
 }: {
   jobTypes: ComboJobTypes[];
+  aggregatorOverride?: string;
 }): Promise<CachedInstitution[]> {
   const config = getConfig();
   const preferences = await getPreferences();
 
-  const supportedAggregators = preferences.supportedAggregators;
+  const supportedAggregators = aggregatorOverride
+    ? [aggregatorOverride]
```

Comment:
If you change the joi validation to only allow prod aggregator names then this should be fine but if you want to keep the sandbox names: `mx_int`, `akoya_sandbox` then you'll probably need to resolve the official aggregator name with this function: https://github.com/Universal-Connect-Project/ucw-app/blob/main/apps/server/src/adapterIndex.ts#L6